pymeasure/instruments/chroma/chroma_63600.py

```
@@ -0,0 +1,522 @@
+        # Auto-discover and add channels
+        for i in range(1,Nslots*2+1):
+            if self.ask(f'CHAN {i};CHAN?') == str(i):
+                # This channel exists, get its id and add it
+                chid = self.ask('CHAN:ID?').split(',')[1]
+                if chid == '63630-80-60':
+                    self.add_child(Chroma63630_80_60,i)
+                elif chid in ['63610-80-20L','63610-80-20R']:
+                    self.add_child(Chroma63610_80_20,i)
+                else:
+                    raise NotImplementedError("Only Chroma 63610-80-20 and 63630-80-60 channels " \
+                                             +"are currently supported.")
```

Flake8 (7.3.0)

[error] continuation line under-indented for visual indent

(E128)